### PR TITLE
Add Captive Portal Scenario 2

### PR DESCRIPTION
**Pull request by strseb**
Adds Scenario 2 of [this figma](https://www.figma.com/file/OQpyNjmowo3BIE23WGnOCM/Captive-Portal---Additional-Messaging?node-id=14%3A1460)

-> This opens the captive-portal alert on detection (when client is connected)
-> clicking the button deactivates the vpn and starts the cp-background monitor :) 

Marking as blocked as this is based on pr #1973

### CODE REVIEW

**Review by lesleyjanenorton — APPROVED** [review]
QML looks good. Is this still blocked? Added @flodolo to review the strings.

File: translations/strings.yaml [hunk 1]
```diff
@@ -244,9 +244,18 @@ customDNSSettings:
 networkSettings:
   multihopTitle: Multi-hop tunnel
 
+
+captivePortalAlert:
+  title: Captive portal detected!
+  header: A captive portal is when the guest Wi-Fi you are using requires you to log in to get internet access. 
+  actionPreActivation: You can turn on your VPN once you have gained internet access by logging in to the portal. 
+  actionPostActivation: You must turn off your VPN in order to see the portal, and turn it back on once you have logged in. 
+  buttonTextPreActivation: Okay
```
Comment (flodolo) [inline translations/strings.yaml:253]: Note: I've seen complains from people about this form vs OK (at least one user on Bugzilla). Not a localization problem, just worth thinking about it ;-)

File: translations/strings.yaml [hunk 2]
```diff
@@ -244,9 +244,18 @@ customDNSSettings:
...
+  buttonTextPostActivation: Turn off VPN
+  
 notifications:
   captivePortalBlockTitle: Guest Wi-Fi portal blocked
-  captivePortalBlockMessage: The guest Wi-Fi network you’re connected to requires action. Click here to turn off VPN to see the portal.
+  captivePortalBlockMessage2: The guest Wi-Fi network you’re connected to requires action. Click to turn off VPN to see the portal.
```
Comment (flodolo) [inline translations/strings.yaml:258]: I mentioned this before: is click the right word for these? This codebase support desktop AND mobile devices.

**Review by flodolo — APPROVED** [review]
It would be great to clarify the "click" bit before merging. The "Okay" it's just a heads-up, I don't have an opinion about it ;-)

File: translations/strings.yaml (hunk 2 above)
Comment (lesleyjanenorton) [reply translations/strings.yaml:258]: @chrisrooneys

**Pull request merged**